src/Engine/ProtoCore/Lang/Replication/Replicator.cs

```diff
@@ -337,7 +337,52 @@ public static List<StackValue> EstimateReducedParams(List<StackValue> formalPara
 
             return reducedParamTypes;
         }
+        
+        /// <summary>
+        /// Compute the effects of the replication guides on the formal parameter lists
+        /// The results of this loose data, and will not be correct on jagged arrays of hetrogenius types
```

Comment:
spelling (lose)

Reply:
copy pasted form the old VM (I only tried to adapt the bare minimum)
We can do spell checks later (There are definitely not the only ones)